packages/generator/tests/acceptance/test_client_generator.py

```diff
@@ -65,36 +65,70 @@ def test___command_line_args___create_client_for_all_registered_measurements___r
     actual_modules = os.listdir(temp_directory)
     assert all(
         [
-            not exc_info.value.code,
+            result.exit_code == 0,
             len(actual_modules) == 2,
             expected_modules == actual_modules,
         ]
     )
 
 
+def test___command_line_args_with_registered_measurements___create_client_using_interactive_mode___renders_without_error(
+    create_client: Callable[[Sequence[str]], Result],
+    test_assets_directory: pathlib.Path,
+    tmp_path_factory: pytest.TempPathFactory,
+    measurement_service: MeasurementService,
+) -> None:
+    temp_directory = tmp_path_factory.mktemp("measurement_plugin_client_files")
+    golden_path = test_assets_directory / "example_renders" / "measurement_plugin_client"
+    filename = "non_streaming_data_measurement_client.py"
+    inputs = [
+        "1",
+        "non_streaming_data_measurement_client",
+        "NonStreamingDataMeasurementClient",
+        "x",
+    ]
+    os.chdir(temp_directory)
+
+    result = create_client(["--interactive"], input="\n".join(inputs))
```

Comment:
```
tests/acceptance/test_client_generator.py:92: error: Unexpected keyword argument "input"  [call-arg]
```

The correct way to type hint this is with a protocol:
```
class CreateClientFunction(Protocol):
    def __call__(self, args: Sequence[str], input: Optional[str] = None): ...
```

```
create_client: CreateClientFunction,
```

Reply:
Thanks for mentioning, I have added updated this by giving a common name `CliRunnerFunction` and used the same for `create_measurement` too.